packages/manager/apps/web-domains/src/alldoms/components/DatagridColumns/DatagridColumnContact.tsx

```diff
@@ -0,0 +1,18 @@
+import { DataGridTextCell } from '@ovh-ux/manager-react-components';
+import React from 'react';
+import { ServiceInfoContactEnum } from '@/alldoms/enum/service.enum';
+import { TServiceInfo } from '@/alldoms/types';
+
+interface DatagridColumnContactProps {
+  contacts: TServiceInfo['customer']['contacts'];
+  contactType: ServiceInfoContactEnum;
+}
+
+export default function DatagridColumnContact({
+  contacts,
+  contactType,
+}: Readonly<DatagridColumnContactProps>) {
+  const contact = contacts.find((c) => c.type === contactType);
```

Comment:
seems kinda weird to me to do this lookup here instead or just passing `contact` to this component and do the lookup in the parent tbh